jekyll/_cci2/github-integration.adoc

```diff
@@ -352,6 +352,12 @@ Once you have set up your first pipeline, you will be able to add triggers to it
 
 NOTE: Only organization admins and users that have admin access to at least one repository in an organization can install the GitHub App.
 
+Once the GitHub App is installed, each user should authorize the GitHub App connection. To do this, each user should navigate to menu:Project Settings[Pipelines>Add Pipeline>Connection] and select btn:[Authorize].
+Authorizing the GitHub App ensures that users can:
+* Trigger pipelines xref:triggers-overview#run-a-pipeline-from-the-circleci-web-app[from the CircleCI web app]. 
+* Run builds, if the flag xref:prevent-unregistered-users-from-spending-credits#[Prevent unregistered user spend] is set to "On". 
+* Run builds, if they require access to xref:contexts#[contexts].
```

Comment:
@yaningo  I incorporated your suggestion but It looks like that didn't work 
![Screenshot 2025-03-17 at 22 04 51](https://github.com/user-attachments/assets/c7e4ff21-364c-40a6-93d2-858e89c84908)

How should I change it for the bullet points to show?

Reply:
Indeed, I noticed that too. I believe it's because of a missing empty line between _"Authorizing the GitHub App ensures that users can:"_ and the actual list.